.github/workflows/draft-release.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         run: |
           RELEASE_NAME=syslog-ng-`cat VERSION`
           sed "1 i${RELEASE_NAME}\n" NEWS.md > /tmp/message
+          mv dbld/build/${RELEASE_NAME}.orig.tar.gz dbld/build/${RELEASE_NAME}.tar.gz
```

Comment:
small note: I did not know at first, the "orig.tar.gz" format comes from the debian packaging method.

Reply:
@lbudai @szemere 

The .orig.tar.gz is an artifact of the debian package building, .tar.gz should be there even without this mv command.

This is our pkg-tarball script (started as a part of the release process):
```
#!/bin/bash

# MODE is passed via the environment

. /dbld/functions.sh

cd /source

SYSLOGNG_DIR=syslog-ng-${VERSION}
SYSLOGNG_TARBALL=${SYSLOGNG_DIR}.tar.gz

cd /build
tar xfz $SYSLOGNG_TARBALL
cp $SYSLOGNG_TARBALL ${SYSLOGNG_TARBALL}.orig
cd $SYSLOGNG_DIR

echo ${VERSION} > VERSION

/dbld/generate-debian-directory $MODE
/dbld/generate-rpm-specfile $MODE

cd ..
tar cfz $SYSLOGNG_TARBALL $SYSLOGNG_DIR

capture_artifacts $SYSLOGNG_TARBALL

```

as you can see the final artifact is called syslog-ng-VERSION.tar.gz

.orig.tar.gz is only used when building the Debian package:
```
function prepare_source() {
    cd /build
    rm -rf syslog-ng-${VERSION}
    tar xf syslog-ng-${VERSION}.tar.gz

    # save the tarball as .orig.tar.gz so that dpkg-buildpackage finds it as the original source
    cp syslog-ng-${VERSION}.tar.gz syslog-ng_${VERSION}.orig.tar.gz
}
```

Again, this is just a copy, but that's the reason that an orig.tar.gz is there. It should be the same as the .tar.gz file (without the orig), but the latter is only present if a .deb is generated.

So, this doesn't seem to be a proper fix for me.

Reply:
@bazsi : unfortunately the `syslog-ng-${VERSION}.tar.gz` has a huge size (54MB) as it contains the generated Debian binaries. 
I think this is a bug (and agree with you that this is a temporary solution that can be fixed after the release is out).
Maybe the best would be to generate only the tarball... the script runs the dbld `release` target which generates packages.
And after package generation the tarball is 'wrong'.